league_rpc_linux/__main__.py

```diff
@@ -35,6 +43,10 @@ def main(cli_args: argparse.Namespace):
         client_id=cli_args.client_id,
         wait_for_discord=cli_args.wait_for_discord,
     )
+
+    p = Process(target=LCU_Thread.start_connector, args=(rpc,))
```

Comment:
also, please name it something more descriptive than "p"

Reply:
renamed to lcu_process